src/sql/pg/create.sql

```diff
@@ -0,0 +1,147 @@
+--
+-- PostgreSQL
+--
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = "warning";
+SET search_path = "public", "pg_catalog";
+--
+-- Sequences
+--
+
+CREATE SEQUENCE "N_CASES_ID_SEQ"
+  INCREMENT 1 
+  START 1
+  CACHE 100;
+
+CREATE SEQUENCE "N_NOTIFICATION_ID_SEQ"
+  INCREMENT 1 
+  START 1
+  CACHE 100;
+
+--
+-- Tables
+--
+CREATE TABLE "N_LU_CASE_STATUS"
+(
+    "CASE_STATUS_ID"             INTEGER NOT NULL ,
+    "CASE_STATUS_CODE"           CHARACTER VARYING(20) NOT NULL ,
```

Comment:
Here is an example of the prefixes I was mentioning earlier. I think we were going to go with calling these just "CODE" and "DESCRIPTION"